src/Security/UserClassBuilder.php

```diff
@@ -179,7 +179,7 @@ private function addGetPassword(ClassSourceManipulator $manipulator, UserClassCo
             // add an empty method only
             $builder = $manipulator->createMethodBuilder(
                 'getPassword',
-                'string',
+                '?string',
                 true,
                 [
                     'This method can be removed in Symfony 6.0 - is not needed for apps that do not check user passwords.',
```

Comment:
Do we even need to add this method anymore? I'm just looking at the comment that we add - maker only supports symfony `6.4`+

If we do not need the method - lets leave this alone and remove the method in a separate PR.